### ISSUE TITLE
Assertion failure for  get-model command

### ISSUE THREAD

**Issue by mudathirmahgoub** [CVC4/CVC4#3188]
Using a static debug build  of CVC4 `./configure.sh --static debug` on the attached file generates the following assertion failure
```
(error "Assertion failure.
CVC4::Expr CVC4::SmtEngine::getValue(const CVC4::Expr&) const
/space/mahgoubyahia/Desktop/CVC4/src/smt/smt_engine.cpp:4235:

  resultNode.getKind() == kind::LAMBDA || resultNode.isConst()
")
```
[alloy.txt](https://github.com/CVC4/CVC4/files/3502341/alloy.txt)

CVC4 --version
```
This is CVC4 version 1.8-prerelease [git master 4924138b]
compiled with GCC version 9.0.1 20190312 (Red Hat 9.0.1-0.10)
on Aug 14 2019 12:54:38
```

**Comment by ajreynol** [comment]
Thanks, I can reproduce, will look into it.

**Comment by ajreynol** [comment]
I believe this is related to https://github.com/CVC4/CVC4/issues/888.

There is a variable `|{"filename":"C://Users//mudat//AppData//Local//Temp//alloy_heredoc16149776506447341045.als","x1":17,"y1":2,"x2":35,"y2":2,"symbolIndex":101}|` that is defined to be equal to a quantified formula. Getting its value causes use to lookup the value of this formula, which apparently does not exist as a SAT literal thus the value of `|{"filename":"C://Users//mudat//AppData//Local//Temp//alloy_heredoc16149776506447341045.als","x1":17,"y1":2,"x2":35,"y2":2,"symbolIndex":101}|` is not constant.

Not sure yet what the best resolution for this is.

**Comment by ajreynol** [comment]
Actually the symbol in question is introduced via a `:named` attribute, so it should not have been appearing in the model in the first place. I've fixed this in #3190.
